src/global/CMakeLists.txt

```diff
@@ -7,4 +7,4 @@ set(global_common_files
 add_library(global_common_objs OBJECT ${global_common_files})
 add_library(global STATIC ${libglobal_srcs}
   $<TARGET_OBJECTS:global_common_objs>)
-target_link_libraries(global ceph-common ${DPDK_LIBRARIES} ${EXTRALIBS})
+target_link_libraries(global common ${DPDK_LIBRARIES} ${EXTRALIBS})
```

Comment:
ah, thanks! right! maybe i should remove ceph-common or common from global's linked libraries. because quite a few executables link against global, this would introduce more code churn. but seems it's unavoidable..